charts/scalar-admin-k8s/templates/scalar-admin-k8s/job.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if not .Values.scalarAdminK8s.cronJob.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "scalar-admin-k8s.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "scalar-admin-k8s.labels" . | nindent 4 }}
+spec:
```

Comment:
Job spec is the same as CronJob's spec.template.spec. If you use `include` to make it common, you can reduce mistakes in future changes.

Reply:
Thank you for the suggestion! Sounds good!
I fixed it in c9d770c331c5af5b1c2d02543588dce14163aa1b .